applicants/README.md

```diff
@@ -0,0 +1,154 @@
+# applicants
+
+## applicants object
+
+```{
+  userId: string,
+  biodata: {
+    firstName: string,
+    lastName: string,
+  },
+  location: {
+    city: string,
+    state: string,
+    country: string,
+  },
+  professional: {
+    institution: string,
+    skills: string,
+  },
+  intro: {
+    introduction: string,
+    funFact: string,
+    forFun: string,
+    whyRds: string,
+    numberOfHours: string,
+  },
+  foundFrom: string,
+  status: string || null,
+  discordLink: string || null
+}
+```
+
+## **Requests**
+
+|                         Route                          |             Description              |
+| :----------------------------------------------------: | :----------------------------------: |
+|                [GET /applications](#get-applications)                |   Returns all the applications in the system   |
```

Comment:
why not modify this to work with query params, in case no query param is passed send back all the data, if it is passed with specific user id/self then give the individual data

Reply:
I don't think that will good according rest principles, I think we should be creating a new endpoint if we are getting any nested resource